build/release/README.md

```diff
@@ -0,0 +1,69 @@
+# Releasing jQuery
+
+This document describes the process for releasing a new version of jQuery. It is intended for jQuery team members and collaborators who have been granted permission to release new versions.
+
+## Prerequisites
+
+Before you can release a new version of jQuery, you need to have the following tools installed:
+
+- [Node.js](https://nodejs.org/) (latest LTS version)
+- [npm](https://www.npmjs.com/) (comes with Node.js)
+- [git](https://git-scm.com/)
+
+## Release Process
+
+1. Ensure all milestoned issues/PRs are closed, or reassign to a new milestone.
+2. Verify all tests are passing in [CI](https://github.com/jquery/jquery/actions).
+3. Run any release-only tests, such as those in the [`test/integration`](../../test/integration/) folder.
+4. Ensure AUTHORS.txt file is up to date (this will be verified by the release script).
+  - Use `npm run authors:update` to update.
```

Comment:
This should also be indented more. Right now, it creates a separate `<ul>` at the same level as the `<ol>` above.

I'd recommend you to look at the rendered HTML and see if the structure makes sense by yourself. You'll uncover all these semantic bugs pretty quickly then.

Reply:
The extra indent always looked like it registered in code as plain text instead of a list item, but it seems to work.

Reply:
The extra indent doesn't work on GitHub.

Reply:
Adding a newline but removing the extra indent seems to work though.